webapp/src/components/sidebar/sidebarBoardItem.tsx

```diff
@@ -35,6 +36,7 @@ const SidebarBoardItem = React.memo((props: Props) => {
         switch (viewType) {
         case 'board': return <BoardIcon/>
         case 'table': return <TableIcon/>
+        case 'gallery': return <ImageIcon/>
```

Comment:
We can fix this later, but let's add a new `GalleryIcon` class, copied from ImageIcon, then assign a new issue to Assad to create a new icon.

Reply:
Done